setup.cfg

```diff
@@ -44,6 +44,8 @@ test =
     appdirs@https://github.com/ActiveState/appdirs/archive/8eacfa312d77aba28d483fbfb6f6fc54099622be.zip
     pytest>=6
     pytest-cov>=2.7
+docs =
+    Sphinx
```

Comment:
always lowerpin dependencies; use sphinx_rtd_theme

Reply:
can we use this? https://github.com/pradyunsg/furo

Reply:
I mean sure 🤷 I'm -0.1 on it until it adds dark them switch button 😆

Reply:
okay nvm then! the work in this PR is done already, but just fyi I much prefer MkDocs 😄

Reply:
I prefer sphinx so all good

Reply:
Still no switch theme button?

Reply:
The sample directories look A LOT better with this theme :sunglasses:

Reply:
> Still no switch theme button?

Button? CSS can detect if the browser is in night mode and apply automatically.

Reply:
Check https://bernat.tech/, also auto detects but there's a button to override. For me furo is not that readable in black so would like to override the auto detected black with white. Just because the OS is in dark mode that doesn't mean my web pages need necessarily to be also in dark mode.

Reply:
I've switched to `sphinx_rtd_theme`.

Honestly, I don't think adding a dark-mode button really makes sense. There's browser plugins to toggle this, or OS-wide settings for those who prefer that. I can't image why a person thinking "I want to read this specific documentation in dark mode, but not every other tab I have open".

Furo also render funky for me in dark mode, for some reason:

![image](https://user-images.githubusercontent.com/730811/126003897-7b5e7eac-c1d3-4ee9-a791-4cc5c68b785b.png)

Reply:
> I can't image why a person thinking "I want to read this specific documentation in dark mode, but not every other tab I have open".

So you're saying my opinion doesn't matter. Some pages render ok in dark, some don't. Why can't I have the choice to make that subjective judgment myself? Why do you want to force that on me?

Reply:
> So you're saying my opinion doesn't matter.

Not at all. In my opinion, every website that has a dark-mode picker should have a font-picker, zoom-picker, font-size-picker, and high-contrast picker.

Fortunately, browsers implement all those features already, saving individual websites having to re-implement in. Why can't you use the controls on your browser (or a plugin if there isn't out out-of-the-box)?

Do you really think it's reasonable for **every single website out there** to implemented the exact same logic, when it's already implemented in the client you're using to render those very websites?

Reply:
Until you convince the browser creators to allow that freedom on a per-page basis via a plugin individual website implementation remains the only option.

Reply:
[Dark Reader](https://darkreader.org/) lets you toggle dark mode on a per-domain basis.